modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/ClassNode.java

```diff
@@ -37,22 +37,10 @@
 import java.util.ArrayList;
 import java.util.BitSet;
 import java.util.Collection;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
 import static org.elasticsearch.painless.WriterConstants.BASE_INTERFACE_TYPE;
-import static org.elasticsearch.painless.WriterConstants.BITSET_TYPE;
 import static org.elasticsearch.painless.WriterConstants.CLASS_TYPE;
-import static org.elasticsearch.painless.WriterConstants.DEFINITION_TYPE;
-import static org.elasticsearch.painless.WriterConstants.DEF_BOOTSTRAP_DELEGATE_METHOD;
```

Comment:
We can remove these constants from the code now right?

Reply:
Yes! Good catch.